azure-app-service/massdriver.yaml

```diff
@@ -0,0 +1,195 @@
+access: private
+type: application
+
+app:
+  envs:
+    # Use jq expressions to build environment variables from input params or connections
+    # LOG_LEVEL: .params.log_level
+    # MONGO_DSN: .connections.mongo_authentication.data.authentication.username + ":" + .connections.mongo_authentication.data.authentication.password + "@" + .connections.mongo_authentication.data.authentication.hostname + ":" + (.connections.mongo_authentication.data.authentication.port|tostring)
+  policies: []
+    # Use jq expressions to select policies from artifact security blocks
+    # - .connections.sqs.data.security.iam.subscribe
+
+params:
+  examples:
+    - __name: Development
+      application:
+        sku_name: S1
+        minimum_worker_count: 1
+        maximum_worker_count: 3
+    - __name: Production
+      application:
+        sku_name: P3v3
+        minimum_worker_count: 1
+        maximum_worker_count: 10
+  required:
+    - application
+    - repo
+    - dns
+  properties:
+    application: 
+      title: Application
+      description: Application configuration
+      type: object
+      required:
+        - location
+        - sku_name
+        - minimum_worker_count
+        - maximum_worker_count
+      properties:
+        location:
+          title: Location
+          description: Azure location
+          type: string
+          $md.immutable: true
+          oneOf:
+            - title: West US
+              const: westus
+            - title: West US 2
+              const: westus2
+            - title: West US 3
+              const: westus3
+            - title: East US
+              const: eastus
+            - title: East US 2
+              const: eastus2
+            - title: North Central US
+              const: northcentralus
+            - title: South Central US
+              const: southcentralus
+            - title: Central US
+              const: centralus
+        sku_name:
+          title: Size
+          description: Azure App Service Plan size
+          type: string
+          oneOf:
+            - title: 1 vCPU, 1.75GB RAM
+              const: S1
+            - title: 2 vCPU 8GB RAM
+              const: P1v3
+            - title: 4 vCPU 16GB RAM
+              const: P2v3
+            - title: 8 vCPU 32GB RAM
+              const: P3v3
+        minimum_worker_count:
+          title: Minimum instances
+          description: Minimum number of instances for autoscale.
+          type: integer
+          minimum: 1
+          maximum: 30
+        maximum_worker_count:
+          title: Maximum instances
+          description: Maximum number of instances for autoscale.
+          type: integer
+          minimum: 1
+          maximum: 30
+        notification_email:
+          title: Notification email
+          description: Receive alerts regarding autoscale events.
+          type: string
+          format: email
+          example: user@example.com
+    repo:
+      title: Repository
```

Comment:
Let's make it clear this is the Docker images settings.  Here are the settings in the `kubernetes-deployment` app: https://github.com/massdriver-cloud/application-templates/blob/main/kubernetes-deployment/massdriver.yaml#L24-L41

Reply:
Updating this block to be similar to linked suggestion.